src/Integration/Language.cs

```diff
@@ -7,10 +7,11 @@
 namespace SonarLint.VisualStudio.Integration
 {
     [DebuggerDisplay("{Name} (ServerKey: {ServerKey}, ProjectType: {ProjectType}, IsSupported: {IsSupported})")]
-    internal sealed class Language : IEquatable<Language>
+    public sealed class Language : IEquatable<Language>
     {
-        public readonly static Language CSharp = new Language("cs", Strings.CSharpLanguageName, ProjectSystemHelper.CSharpProjectKind);
-        public readonly static Language VBNET = new Language("vbnet", Strings.VBNetLanguageName, ProjectSystemHelper.VbProjectKind);
+        public readonly static Language Unknown = new Language(Strings.UnknownLanguageName);
```

Comment:
Introduced an 'Unknown' language type, mainly to allow `Language.ForProject` to always have something to return, other than `null`. Makes life a little easier.